content/connect/index.textile

```diff
@@ -0,0 +1,263 @@
+
+This feature is intended for debugging, so once the number of new connections exceeds the number of messages per second permitted by the lifecycle channel, new events will be dropped. This means if you want a definitive list of everyone using your app you'd be best using "token authentication":/auth/token to create your own 'auth server'.
+
+h3(#metachannels). Connection metachannels
+
+Sampled connection events for an app are published to the channel @[meta]clientEvents:connections@. Connection events can be used to compile statistics relating to client population.
+
+The connection event types are:
+
+- connection.opened := The connection was successfully made by the client to Ably.
+- connection.closed := The connection was explicitly closed by the client.
+- connection.refused := The connection was rejected for an expected reason, such as invalid credentials or a malformed request.
+- connection.failed := The connection was rejected for an unexpected reason, such as a network failure.
+
+Events contain a subset of the following metadata in the @data@ field:
+
+- host := The host the connection was made to.<br>__Type: @String@__
+- requestId := The unique ID of the request that Ably can use to correlate a connection event with internal logs, if necessary.<br>__Type: @String@__
+- region := The region the connection was made from.<br>__Type: @String@__
+- headers := The headers sent with the connection.<br>__Type: @JSON Object@__
+- query := The parsed query string of the connection request, excluding authentication parameters. It contains connection information such as the SDK version and any custom transport parameters.<br>__Type: @JSON Object@__
+- connectionId := The unique ID of the connection.<br>__Type: @String@__
+- clientId := The ID of the client that attempted the connection.<br>__Type: @String@__
+- channels := A list of channels included in the request. This is only relevant where channels are supported as part of the connection request, such as with "SSE":/protocols/sse. <br>__Type: @Array@__
+- duration := The duration the connection was open for.<br>__Type: @Integer@__
+- error := The details of any error encountered whilst making the connection request. It includes an error message, error code and HTTP status code.<br>__Type: @JSON Object@__
+
+An example of a @connection.closed@ event is:
+
+```[json]
+{
+  "host": "realtime.ably.io",
+  "requestId": "fbbcb0ab-fa56-47c4-bbd4-fccc22a271b8",
+  "region": "us-east-1",
+  "headers": {
+    "host": "realtime.ably.io",
+    ...
+  },
+  "query": {
+    "format": "json",
+    "heartbeats": "true",
+    "v": "1.2",
+    "lib": "js-web-1.2.9"
+  },
+  "connectionId": "54321",
+  "clientId": "12345",
+  "duration": 61151
+}
+```
+
+p(note). The Ably dashboard contains a developer console. In the developer console you can view connection events. See the "developer console section":/account/LINK-ME for more information.
```

Comment:
This should be a caveat box about connections across multiple browser tabs (source [here](https://faqs.ably.com/can-a-single-connection-be-shared-across-multiple-browser-tabs)).